docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,69 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** copies container images between hosts on a network. You can load to the remote host or from the remote host as well as in between two remote hosts.
+Note: `::` is used to specify the output file on the host or the image name depending on if you are saving or loading.
```

Comment:
I don’t think this is actually true - or is it? If it is, it shouldn’t be; automatically choose a temporary file name, use it, and delete it, so that the user doesn’t have to care.

Reply:
you're right, in the first version we allowed specifying of the file but not anymore so I will remove.

Reply:
There is still a ::FILELOCATION below.